client/src/components/AlsoViewedItemsCard.jsx

```diff
@@ -121,17 +122,16 @@ class AlsoViewedItemsCard extends React.Component {
   componentDidMount() {
     const { categoryId } = this.state;
 
-    fetch(`http://localhost:3004/api/alsovieweditems/categoryid/${categoryId}`)
-      .then((response) => {
-        if (response.ok) {
-          return response.json();
-        }
-        throw new Error('Something went wrong....');
-      })
-      .then(data => this.setState({
-        alsoViewedItems: data, totalPages: calculatePages(data.length),
-      }))
-      .catch(console.log);
+    $.ajax({
```

Comment:
Why did you stop using `fetch()`?

Reply:
To overcome jest-enzyme test. Otherwise option is to mock fetch.